code/modules/power/power.dm

```diff
@@ -330,7 +330,7 @@
 	else if(!power_source)
 		return 0
 	else
-		log_admin("ERROR: /proc/electrocute_mob([M], [power_source], [source]): wrong power_source")
+		log_admin("ERROR: PROC_REF(electrocute_mob([M]), [power_source], [source]): wrong power_source")
```

Comment:
Again, verify your regex replacements.

For those reading this, this is why I am so averse to regex replacements, especially if you dont verify it first.